### PR TITLE
Paas 1870

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-1870

Short description:

### CODE REVIEW

File: assets/common/hn_upgrade.py [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+        self.send_post_request(url, headers=headers)
+
+    def send_post_request(self, url, data=None, headers=None):
+        try:
+            response = requests.post(url=url, headers=headers, data=data)
+            if response.status_code != HTTPStatus.OK:
+                logger.error(str(response.status_code) + " : " + response.text)
+                exit(1)
+        except requests.RequestException as exception:
+            logger.error("Exception when trying to send the GET request" + str(exception))
```
Comment (adoux) [inline assets/common/hn_upgrade.py:249]: POST *

File: assets/common/hn_upgrade.py [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+
+def argparser():
+    """script options"""
+    parser = argparse.ArgumentParser()
+
+    args_list = parser.add_argument_group('Arguments')
+    args_list.add_argument("--dry-run",
+                           action="store_true",
+                           dest="dry_run",
+                           help="If set, do not perform any actions, only outputs what it would have done")
```
Comment (adoux) [inline assets/common/hn_upgrade.py:303]: does *

**Comment by adoux** [comment]
Sorry I'm being a Kevin here but you did a perfect job, don't have any other change request than wordings.

Good job 👍

**Comment by Ronan-Gorain** [comment]
Changes done. I also renamed the two firsts commits that still had the old ticket name (COPS-XXX)

**Review by adoux — APPROVED** [review]

**Pull request merged**